src/store/graphql/variants/models.ts

```diff
@@ -0,0 +1,62 @@
+export enum Impact {
+  High = 'HIGH',
+  Moderate = 'MODERATE',
+  Low = 'LOW',
+  Modifier = 'MODIFIER',
+}
+
+export type Consequence = {
+  node: {
+    symbol: string;
+    consequences: string[];
+    vep_impact: Impact;
+    canonical?: boolean;
+    aa_change: string | undefined;
+    [key: string]: any;
+  };
+  [key: string]: any;
+};
+
+export type ClinVar = {
+  clinvar_id: string | undefined;
+  clin_sig: string | undefined;
```

Comment:
`?.` and `T | undefined` is exactly the same definition. We should be consistant accros the project yet the same file.